spec/models/agents/wunderlist_agent_spec.rb

```diff
@@ -54,8 +54,9 @@
 
   describe "#receive" do
     it "send a message to the hipchat" do
-      stub_request(:post, 'https://a.wunderlist.com/api/v1/tasks').with { |request| request.body == 'abc'}
+      stub_request(:post, 'https://a.wunderlist.com/api/v1/tasks')
       @checker.receive([@event])
+      expect(WebMock).to have_requested(:post, "https://a.wunderlist.com/api/v1/tasks")
```

Comment:
I didn't see any indication that the `.with { |request| request.body == 'abc'}` part was doing anything, or that that boolean expression had any effect.